README.md

```diff
@@ -51,4 +54,4 @@ This pipeline and the forked powerdns bosh release currently only support a sing
 Verification is done automatically via the Concourse pipeline.  To test DNSSEC locally:
 * Retrieve root keys: `dig . DNSKEY @8.8.8.8 | grep -Ev '^($|;)' > root.keys`
 * Check RRset: `dig +sigchase +dnssec +trusted-key=./root.keys example.com. A @YOUR_PUBLIC_DNS_IP | grep -P "^;; VERIFYING A RRset for example.com. with DNSKEY:\d+: success$"`
-* Check DS: `dig +sigchase +dnssec +trusted-key=./root.keys example.com. A @YOUR_PUBLIC_DNS_IP | grep 'DNSSEC validation is ok: SUCCESS'`
+* Check DS: `dig +sigchase +dnssec +trusted-key=./root.keys example.com. A @8.8.8.8 | grep 'DNSSEC validation is ok: SUCCESS'`
```

Comment:
This was YOUR_PUBLIC_DNS_IP since that could be done even without updating registrar settings, should we explain it further?

Reply:
It won't work for the DS key because again, DS is upstream of these servers. So we can yank this entirely, or simply not look for SUCCESS here and explain what to look for.